gap/PGMatrix.gd

```diff
@@ -0,0 +1,96 @@
+#!
+#!
+#! @Chapter Hyperbolic lattice symmetries
+
+#!
+#! @Section Point group matrices
+#! @SectionLabel PGMatrix
+#!
+#! Symmetry elements of triangle groups might act non-trivially on hyperbolic momenta. 
+#! As such, the degrees of freedom of observables and topological invariants, like the 
+#! first Chern numbers in momentum space, might be contstrained by the hyperbolic lattice 
+#! symmetries of underlying models. We can determine how hyperbolic momenta transform as 
+#! hyperbolic lattice symmetry transformations act on Abelian states. This can be extracted 
+#! by constructing point-group matrices.
+
+#! @Description
+#!   Construct point-group matrices for hyperbolic lattice <A>symmetries</A>, given as words in 
```

Comment:
In case it makes sense to restrict to point-group elements, would the input still be in terms of elements of $\Delta$? Or rather $G$? Maybe $\Delta$ makes more sense because then one does not need to care about homomorphisms when going to supercells...